Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm

```diff
@@ -409,8 +409,13 @@ static void updateCustomAppearance(CALayer *layer, GraphicsLayer::CustomAppearan
 #if PLATFORM(IOS_FAMILY)
 void RemoteLayerTreePropertyApplier::applyPropertiesToUIView(UIView *view, const LayerProperties& properties, const RelatedLayerMap& relatedLayers)
 {
-    if (properties.changedProperties.containsAny({ LayerChange::ContentsHiddenChanged, LayerChange::UserInteractionEnabledChanged }))
-        view.userInteractionEnabled = !properties.contentsHidden && properties.userInteractionEnabled;
+    if (properties.changedProperties.containsAny({ LayerChange::ContentsHiddenChanged, LayerChange::UserInteractionEnabledChanged })) {
+        bool userInteractive = !properties.contentsHidden && properties.userInteractionEnabled;
+        if ([view isKindOfClass:[WKChildScrollView class]])
+            [(WKChildScrollView *)view setScrollEnabled:userInteractive];
```

Comment:
```suggestion
        if (auto scrollView = dynamic_objc_cast<WKChildScrollView>(view))
            [scrollView setScrollEnabled:userInteractive];
```

Similar comment in two other places.

Reply:
This one is fixed (and consistent with the rest of the file), but `RemoteLayerTreeViews.mm` is _full_ of `isKindOfClass`, are we progressively migrating file by file?

Reply:
I don't think there's a concerted effort to migrate – it's just preferred since it removes the redundant class name. Also, there are probably many people working on t WebKit that aren't aware of `dynamic_objc_cast`.

I'd be okay if you left `RemoteLayerTreeViews.mm` as-is though.